pkg/controller/volume/expand/expand_controller.go

```diff
@@ -111,6 +100,7 @@ func NewExpandController(
 		pvcsSynced: pvcInformer.Informer().HasSynced,
 		pvLister:   pvInformer.Lister(),
 		pvSynced:   pvInformer.Informer().HasSynced,
+		queue:      workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "expand"),
```

Comment:
can we call it volume_expand just to be a little more specific compared to all the workqueues in controller manager

Reply:
fixed